doc/02-Installation.md.d/From-Source.md

```diff
@@ -0,0 +1,22 @@
+# Installing Icinga Notifications Web from Source
+
+Please see the Icinga Web documentation on
+[how to install modules](https://icinga.com/docs/icinga-web-2/latest/doc/08-Modules/#installation) from source.
+Make sure you use `notifications` as the module name. The following requirements must also be met.
+
+### Requirements
+
+- PHP (≥7.2)
+- PHP needs the following extensions to be installed and activated:
+    - `cURL`
+    - `dom`
+    - `json`
+    - `libxml`
```

Comment:
Is this so? Have you verified each?

Reply:
No, I kept it from the IcingaDB module as we're using a very similar stack.
We also do not have any mentions about PHP extensions in our `module.info` nor a `composer.json` file which would usually list the needed PHP extensions used by the project.

The only extensions which I surely know that are used would be `json` and `dom`.

Reply:
json sure, but dom?

cURL and libxml definitely not, just checked.